client/app/telecom/telephony/service/fax/filtering/telecom-telephony-service-fax-filtering.controller.js

```diff
@@ -51,7 +64,25 @@ angular.module("managerApp").controller("TelecomTelephonyServiceFaxFilteringCtrl
             billingAccount: $stateParams.billingAccount,
             serviceName: $stateParams.serviceName
         }, _.pick(self.screenListsForm, "filteringList")).$promise.catch(function (err) {
-            return new ToastError(err);
+            Toast.error(err);
```

Comment:
Can you provide an error message ? Something like `[$translate.instant("blabla"), _.get(err, "data.message")].join(" ");` ?